README.md

```diff
@@ -49,15 +53,57 @@ chmod +x ./deployment/build-s3-dist.sh
     s3://my-bucket-name-$REGION/$SOLUTION_NAME/$VERSION/
     ``` 
 
-## Deploying the customized solution
+### Deploying the customized solution
 * Get the link of the custom-control-tower-initiation.template loaded to your Amazon S3 bucket. 
 * Deploy the Customizations for AWS Control Tower solution to your account by launching a new AWS CloudFormation stack using the link of the custom-control-tower-initiation.template.
 
+## Deploying with GitHub as the Source
+
+### Prepare a GitHub Repository
+Create a repository within your GitHub account, and populate with the contents of this repository. 
```

Comment:
Why do they need  complete content of this repository? Isn't this GitHub repo meant for source of customizations?

Reply:
Good point, updated.